app-preprod.yaml

```diff
@@ -38,9 +51,7 @@ spec:
       memory: 1Gi
       cpu: 200m
   ingresses: # Optional. List of ingress URLs that will route HTTP traffic to the application.
-    - https://familie-integrasjoner.nais.preprod.local # Deprecated! Erstattes av https://familie-integrasjoner.dev.adeo.no
-    - https://familie-integrasjoner.dev-fss.nais.io # Deprecated! Erstattes av  https://familie-integrasjoner.dev.adeo.no
-    - https://familie-integrasjoner.dev.adeo.no
+    - https://familie-integrasjoner.teamfamilie.dev.adeo.no
```

Comment:
Jeg tror vi må beholde de andre til att de har gått over til den nye

Reply:
alle bruker vel service discovery(app-navn)? finner ikke spor av disse urlene i github under nav orgen :)

Reply:
Blir ikke service discovery url http://familie-integrasjoner.teamfamilie når man endrer namespace? Slik som saf.q2

Reply:
jo, tanken var å rulle ut denne endringen i teamfamilie namespacet og teste at det virker sammen med saksbehandlingsløsningne før vi sletter appen i default namespace, så i en liten periode vil integrasjoner kjøre i 2 namespace mens vi endrer konfig for andre apper. Tar gjerne imot innspill til om det finnes andre måter å komme over til teamnamespace uten nedetid :)

Reply:
Aah! Skjønner! Visste ikke namespace ble tatt med i url når man endret namespace.
https://doc.nais.io/clusters/team-namespaces/#integration-via-kubernetes-service-discovery denne burde vel virke! Har ikke testet den selv :) 

Husk att gateways fra sbs må routes over til ny namespace også https://github.com/navikt/familie-ef-soknad-api/blob/master/src/main/resources/application.yml#L16
Brukes for kodeverk